vendor/forms.py

```diff
@@ -387,6 +389,30 @@ def clean(self):
 
         return cleaned_data
 
+
+class SiteSelectForm(forms.Form):
+    site = forms.ModelChoiceField(queryset=Site.objects.all())
+
+
+class SubscriptionForm(forms.Form):
+    site = forms.ModelChoiceField(queryset=Site.objects.all())
+    customer_profile = forms.ModelChoiceField(queryset=None)
+    subscription_id = forms.CharField(max_length=30)
+    transaction_id = forms.CharField(max_length=30)
+    offer = forms.ModelChoiceField(queryset=None)
+    
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        if 'site' in self.initial:
```

Comment:
do we want to set a default for site, since we're querying using it and the variable is only set behind an if statement? or throw an error if site is not in initial or something like that?

Reply:
added the error, thanks for the suggestions